src/content/core/user-roles-and-permissions.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: User Roles and Permissions in Okteto
+description: Understand User Roles and Permissions in the Okteto Platform
+sidebar_label: User Roles and Permissions
+id: user-roles-and-permissions
+---
+
+
+The Okteto platform offers a robust Role-Based Access Control (RBAC) system to manage and control access to different environments within the platform.
+This page outlines the roles, permissions, and key features available to Admins and Developers in Okteto.
+
+## User Roles
+
+In Okteto, there are two primary roles: **Admins** and **Developers**.
+
+## Admins
+
+Admins have extensive control and oversight capabilities across the Okteto platform. Key features include:
+
+### Admin Access to Any Environment
+Admins can perform all operations on any Namespace or Preview Environment, including:
+- Deleting, sharing, waking, sleeping, and marking Namespaces or Preview Environments as persistent
+- Deploying or deleting applications using both Okteto and `kubectl`
+- Accessing all resources and sub-resources within any Development Environment
+- [Transferring Namespace ownership](core/namespaces.mdx#transfer-namespace-ownership) between developers
+- Fine tuning Developer access to Namespaces and Preview Environments by configuring [the `serviceAccounts` section within the Okteto Helm configuration](self-hosted/helm-configuration.mdx#serviceaccounts).
+
+
+**Benefits**: This level of access allows admins to manage and troubleshoot environments, even if the original owner is unavailable.
+
+## Access to the Okteto Admin Dashboard
```

Comment:
Should this have another `#`? It seems at the same level that `Admins` and `Developers` but I think we wanted it to be part of `Admins`